src/test_single_prefill.cu

```diff
@@ -34,7 +34,7 @@ void _TestSinglePrefillKernelCorrectness(size_t qo_len, size_t kv_len, size_t nu
   std::vector<DTypeOut> o(qo_len * num_qo_heads * head_dim);
 
   utils::vec_normal_(q);
-  utils::vec_normal_(k);
+  utils::vec_fill_<DTypeIn>(k, 1.f);
```

Comment:
This change is just for debugging and we should use more rigorous normal initialization
```suggestion
  utils::vec_normal_(k);
```